daml-lf/language/daml-lf.bzl

```diff
@@ -1,8 +1,6 @@
 # Copyright (c) 2020 The DAML Authors. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-lf_stable_version = "1.7"
+lf_stable_version = "1.8"
 lf_latest_version = "1.8"
 lf_dev_version = "1.dev"
-
-lf_versions = [lf_stable_version, lf_latest_version, lf_dev_version]
```

Comment:
Why don't we need this anymore?

Reply:
We didn’t need it before either but it makes even less sense now. It doesn’t include all versions supported by the compiler, it would include duplicates, …. So given that it’s unused just dropping it seemed like the most sensible option.